boards.txt

```diff
@@ -990,6 +990,7 @@ esp32wrover.menu.PartitionScheme.defaultffat=Default 4MB with ffat (1.2MB APP/1.
 esp32wrover.menu.PartitionScheme.defaultffat.build.partitions=default_ffat
 esp32wrover.menu.PartitionScheme.default_8MB=8M with spiffs (3MB APP/1.5MB SPIFFS)
 esp32wrover.menu.PartitionScheme.default_8MB.build.partitions=default_8MB
+esp32wrover.menu.PartitionScheme.default_8MB.upload.maximum_size=3342336
```

Comment:
looks like the `fatflash` option needs a similar override as `2097152`

Reply:
I have missed this one. Thanks @atanisoft for pointing that out. Will add it to the PR :)